pkg/snapshotter/snapshotter_test.go

```diff
@@ -68,11 +68,8 @@ func createMockServer(t *testing.T) (*gomock.Controller, *driver.MockCSIDriver,
 func TestCreateSnapshot(t *testing.T) {
 	defaultName := "snapshot-test"
 	defaultID := "testid"
-	createTimestamp := ptypes.TimestampNow()
-	createTime, err := ptypes.Timestamp(createTimestamp)
-	if err != nil {
-		t.Fatalf("Failed to convert timestamp to time: %v", err)
-	}
+	createTimestamp := timestamppb.Now()
+	createTime := createTimestamp.AsTime()
```

Comment:
Same as above.

Reply:
Response same as above, AsTime() doesn't return a nillable type or an error to check.